docs/design.md

```diff
@@ -62,18 +62,23 @@ Plugins that allow you to perform followings per protocols
    
 
 ### 3. Monitoring
-The blockchain protocols have so many unique logs, and it brings a lot of data which causes difficulties in finding meaningful data by standardizing it to make it easier to view.
-The most validator teams have trouble managing logs from running nodes due to log's varieties.
-**VATZ**'s monitoring service is designed to find a way to manage all logs from nodes efficiently with minimum cost.
+The blockchain protocols have so many unique logs, and it brings a lot of data which causes difficulties in finding meaningful data by standardizing it to make it easier to view.<br/>
+The most validator teams have trouble managing logs from running nodes due to log's varieties.<br/>
+**VATZ**'s monitoring service is designed to find a way to manage all logs from nodes efficiently with minimum cost.<br/>
 
-We are targeting for followings:
+**AS-IS** <br/>
+![monitoring-as-is](../docs/images/monitoring_as_is.png)
 
-> 1. Manage **VATZ** with Dashboard  (2022-Q3)
-> 2. Unified Log exporter (2023-Q2)
+Currnetly VATZ supported to send metrics like liveness of plugins for promethues. Metrics may be added in the future.
```

Comment:
I prefer to update this sentence as 

VATZ currently supports sending metrics for followings for Prometheus: 
- `VATZ` up 
-  `VATZ` plugins liveness 

More metrics will be available in the future.

Reply:
@xellos00 updated it!
- Commit: 009e7c186cf3b4c77a352a1ee17165c707dd7c33